assets/scss/hero.scss

```diff
@@ -82,3 +129,24 @@
     }
   }
 }
+
+.exhibition-page {
+  h1 {
+    font-size: 2.375rem;
+    margin-top: 2.5rem;
+  }
+  p.lead {
+    font-size: 1.5rem;
+    color: $slategrey;
+  }
+  .hero-wrapper {
+    max-width: 51.25rem;
+  }
+  .jumbotron:after {
+    display: none;
+  }
+  article, section p {
+    max-width: 45rem;
+    margin: 0 auto;
```

Comment:
This has some unwanted side effects. Go for example to /en/exhibitions/the-silk-and-the-blood/ravenna and look for the heading "City of mosaics", there is no longer space between the previous paragraph and this heading.